### ISSUE TITLE
Use booleans to avoid warnings

### ISSUE THREAD

**Issue by ggbioing** [learnables/learn2learn#99]
https://github.com/learnables/learn2learn/blob/22dfb50aa5557162919f361f24df8353ec3441e4/examples/vision/meta_mnist.py#L81-L84
Change to:
```
adaptation_indices = torch.zeros(data.size(0)).bool()
adaptation_indices[torch.arange(shots*ways) * 2] = True
adaptation_data, adaptation_labels = data[adaptation_indices], labels[adaptation_indices]
evaluation_data, evaluation_labels = data[~adaptation_indices], `labels[~adaptation_indices]
